DNN Platform/Library/Entities/Urls/AdvancedUrlRewriter.cs

```diff
@@ -2338,8 +2338,9 @@ private static bool IgnoreRequestForInstall(string physicalPath, string refererP
 
         private static bool IgnoreRequestForWebServer(string requestedPath)
         {
-            if (requestedPath.ToLowerInvariant().IndexOf("synchronizecache.aspx", StringComparison.Ordinal) > 1
-                || requestedPath.EndsWith("keepalive.aspx", true, CultureInfo.InvariantCulture))
+            //Should standardize comparison methods

```

Comment:
One uses EndsWith, the other looks for it anywhere.  I think both could be safe in the future with ends with

Reply:
I was assuming that `synchronizecache` might have parameters, whereas `keepalive` is not expected to

Reply:
Correct, synchronizecache always has params. Keepalive shouldn't, but it might since some services add a random number param on requests to avoid caching. I think changing it to indexof would be best.